.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - "10"
+
+cache:
+  directories:
+    - "node_modules"
+
+script: npm test
```

Comment:
Just a question out of curiosity, is it gonna run just tests? Not better add the linting here as well?

Reply:
Oh nevermind, we don't have a lint script :p